specification/glossary.md

```diff
@@ -0,0 +1,44 @@
+# Glossary
+
+Below are a list of some OpenTelemetry specific terms that are used across this
+specification.
+
+## (Telemetry) Library
+
+Denotes the actual *OpenTelemetry* library in use.  
+See [Library](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-resource-semantic-conventions.md#library)
+
+<a name="instrumented_library"></a>
+
+## Instrumented Library
+
+Denotes the library instrumented with OpenTelemetry API calls, thus the library
```

Comment:
This definition sounds a lot like the definition for the "instrumenting library." I would rather see something along the lines of "Denotes the library which is being monitored/observed." In many cases, this library will not be the one with the API calls, because it is monkey patched.

Reply:
Agreed. Let me rephrase that.

Reply:
"The library that is producing telemetry data" sounds to me almost certainly like the instrumenting library. The instrumented library is the one about which the data is produced imho.